command/oooo.go

```diff
@@ -1,21 +1,77 @@
 package command
 
+import (
+	"errors"
+	"strings"
+)
+
+func isVowel(char byte) bool {
+	switch char {
+	case 'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U':
+		return true
+	default:
+		return false
+	}
+}
+
 type OoooCommand struct{}
 
 func (oc *OoooCommand) Name() string {
 	return "oooo"
 }
 
+// OOOO-ify a provided string
```

Comment:
It's standard for function comments in Go to begin with the name of the function
```suggestion
// Handle OOOO-ify a provided string
```